apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/service_name/Servlet2PlusInstrumentation.java

```diff
@@ -52,7 +54,7 @@
  * Determines the service name based on the webapp's {@code META-INF/MANIFEST.MF} file early in the startup process.
  * As this doesn't work with runtime attachment, the service name is also determined when the first request comes in.
  */
-public abstract class InitServiceNameInstrumentation extends AbstractServletInstrumentation {
+public abstract class Servlet2PlusInstrumentation extends AbstractServletInstrumentation {
```

Comment:
We only support Servlet 3+
```suggestion
public abstract class InitServiceNameInstrumentation extends AbstractServletInstrumentation {
```